packages/beacon-node/src/chain/bls/multithread/jobItem.ts

```diff
@@ -70,15 +69,15 @@ export function jobItemWorkReq(job: JobQueueItem, format: PointFormat, metrics:
       // and not a problem in the near future
       // this is monitored on v1.11.0 https://github.com/ChainSafe/lodestar/pull/5912#issuecomment-1700320307
```

Comment:
the comment is not relevant anymore, need to update it with test result on feat2

Reply:
`aggregateWithRandomness` now takes ~2s vs ~1.78s on stable mainnet node, just something to note

<img width="1328" alt="Screenshot 2024-07-23 at 10 01 47" src="https://github.com/user-attachments/assets/2cc9b410-a51a-465c-a32b-242d5d2dd240">